pkg/telemetry/metrics.go

```diff
@@ -5,6 +5,7 @@ import (
 	"time"
```

Comment:
We add here the two new interfaces for the metrics, and their corresponding structs for deserialization.